subprojects/gradle/runner/service/src/main/kotlin/com/avito/runner/service/worker/device/adb/listener/AdbDeviceEventsListener.kt

```diff
@@ -0,0 +1,41 @@
+package com.avito.runner.service.worker.device.adb.listener
+
+import java.nio.file.Path
+
+interface AdbDeviceEventsListener {
```

Comment:
In the Listener convention, we should add the `on` prefix to each function

Reply:
thanks, added